packages/app/components/Nav/NavBar.tsx

```diff
@@ -0,0 +1,53 @@
+import styled from "styled-components";
+import React, { Component, useState } from "react";
+import LeftNavBar from "./LeftNavBar";
+import { Drawer, Button } from "antd";
+import RightNavBar from "./RightNavBar";
+
+const Container = styled.div`
+  width: 1440px;
+  height: 64px;
+`;
+interface NavBarProps {
+  course: Course;
+}
+
+export default function NavBar({ course }: NavBarProps) {
```

Comment:
Looks like the course is coming in as props but is not used in the component. I think you'll want to use `course.name` where is says "Course" on line 29

Reply:
I left this unfinished because I thought that there would need to be a drop down added to the course name, so when you clicked on it the list of your courses would appear. There wasn't a course data definition in there when I did the pr so I couldn't use the field. I can go back and change it now that I have the version with the data definitions.

Reply:
I think this should probably be done in a separate PR, so I'm going to leave it for now?

Reply:
Yeah that's all good